<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabAnalysis</class>
 <widget class="QWidget" name="TabAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="14" column="1">
      <widget class="QLabel" name="labelModelInfo">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineModelname">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushUpdate">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Analysis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="comboAnalysis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineAnalysis_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushNewAnalysis">
         <property name="text">
          <string>Add new</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QListView" name="listCellid">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QListView" name="listModelname">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="lineAnalysis"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Model search</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBatch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBatchName">
         <property name="text">
          <string>Current batch name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushPreview">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushView">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineCellid">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Batch</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cell search</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushScatter">
         <property name="text">
          <string>Scatter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushBar">
         <property name="text">
          <string>Bar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushPareto">
         <property name="text">
          <string>Pareto</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="0">
      <widget class="QPushButton" name="pushRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QTableWidget" name="tableModels"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
